src/app/page.js

```diff
@@ -1,8 +1,9 @@
 import Landing from "@/components/home/Landing";
-
+import Event from "@/components/events/Event";
 const home = () => {
   return (
     <div>
+      <Event />
```

Comment:
it's ok to have this on the home page to test your issue but make sure you take it out before you commit

Reply:
Oh, I am so sorry for this.  Please check it again. Next time I gonna check it carefully. Thank you!